src/Jobs/Invite.php

```diff
@@ -88,18 +90,44 @@ public function handle()
      */
     private function inviteUserIntoGuild()
     {
-        $new_nickname = optional($this->discord_user->group->main_character)->name;
 
-        $user = app('discord')->guild->addGuildMember([
-            'user.id'      => $this->discord_user->discord_id,
-            'guild.id'     => intval(setting('warlof.discord-connector.credentials.guild_id', true)),
-            'nick'         => ! is_null($new_nickname) ? $new_nickname : $this->discord_user->nick,
-            'access_token' => $this->getAccessToken(),
+        $corp_id = $this->discord_user->group->main_character->corporation()->corporation_id;
+        $corp = CorporationInfo::where('corporation_id', $corp_id)->first();
+        $ticker = "[".$corp->ticker."]";
+        $new_nickname = $this->discord_user->group->main_character->name;
```

Comment:
Can you make the ticker optional by appending some settings in the connector ?
So we still allow both `[ticker] character` and `character` - it's up to the end user :)